first-analysis-steps/analysisflow.md

```diff
@@ -0,0 +1,102 @@
+Giving you tools to get those samples is the main goal of the first analysis steps lessons.  
+
+{% callout "Reminder from the data flow lesson" %}
+
+The output of the last stages of the [LHCb data flow](dataflow), namely stripping (Run 1 and Run 2), Turbo stream (Run 2), sprucing (Run 3) and Run 3 HLT2 lines can be accessed by user. 
+Selections made at these last stages are also defined by users. 
+Their output is saved to disk and later is used to produce a data file that will contain only the information that interests users.
+This is done using the software package called DaVinci.
+
+{% endcallout %}
+
+### Getting data files
+
+After preselecting data either in the stripping, sprucing or triggering step, user can produce a data file, called ntuple by running DaVinci package.
+The ntuple is a data file that contains information about events or candidates in the data sample, like mass of a candidate or trigger decision flags.
+For the data sample information is coming from reconstruction, but for the simulation sample you can request both the reconstructed values of observables/variables or generator-level values.
+The reconstructed values are the ones that passed the entire reconstruction pipeline with detector effects included and generator-level values are the original values that the event generator has produced.
+This allows to study different reconstruction and detector effects for your analysis.
+
+DaVinci can be run using CERN grid computing system directly via ganga or via Analysis Productions. 
+Using gange will give you more control over the jobs (little scrips and macros you run on the remote machines) and also will allow you to do way more than just running DaVinci. 
+Analysis Productions, however, are more user-friendly and provide
+Analysis Productions are also extremely important for the preservation of your analysis.
+We will discuss the concept of analysis preservation a bit later in this lesson.
+Unless you want to run some really unusual ntuple jobs on ganga, you should always go for Analysis Productions. 
+
+In first analysis steps we cover both running DaVinci on [ganga](https://lhcb.github.io/starterkit-lessons/first-analysis-steps/davinci-grid.html) and via [Analysis Productions](https://lhcb.github.io/starterkit-lessons/first-analysis-steps/analysis-productions.html).
+
+
+### Useful high energy physics analysis tools
+
+After getting the ntuples a user usually develops new analysis code or expands an existing code, that their collaborators use. 
+Analysis code is usually based on the popular high-energy physics software tools or on the more general data analysis tools, like [numpy](https://numpy.org/) or [pandas](https://pandas.pydata.org/). 
+
+{% callout "Popular data analysis tools in HEP" %}
+
+There are multiple packages that collaboratos use in LHCb.
+Some of this packages are CERN supported, some are projects from the LHCb collaborators and some are supported by external community.  
```

Comment:
to not offend anyone (🤪) it would be good to say something like "this list is by no means exhaustive"

Reply:
Good point 😅